### DISCUSSION TITLE
CLI to pair by OTP

### DISCUSSION THREAD

**Discussion by ShatsDev** [ClassicOldSong/Apollo#136] [Q&A]
Sunshine provides CLI command to pair device by chosen PIN.
Apollo supports OTP - greate feature.
I am looking for CLI command to get Passphrase and PIN for OTP.

When needed my PC will run this CLI command automatically to get the Passphrase and PIN, this info will be sent to some destination by the network. (I know to read Sunshine console output programatically). 
So all what I need is CLI command to get Passphrase and PIN.

**Comment by ClassicOldSong** [comment]
You can emulate login and then get OTP from http config API.
